components/Cta/CtaNextLink.tsx

```diff
@@ -32,27 +33,14 @@ export const CtaNextLink = React.forwardRef<HTMLAnchorElement, CtaNextLinkProps>
     className,
     ...rest
   } = props;
-
-  // Normalize the href and strip the slug prefix.
-  const prefix = getSlugPrefix();
-  const path = href.toString();
-  const hrefParts = path.split('/');
-
-  // Remove empty strings from the array.
-  const cleanParts = hrefParts.filter((s:string) => s.length);
-
-  // If the first part of the URL is the slug prefix, remove it.
-  if (cleanParts[0] === prefix) {
-    cleanParts.shift();
-  }
-
-  const strippedHref = `/${cleanParts.join('/')}`;
+  const strippedHref = sbStripSlugURL(href);
+  const processedHref = strippedHref.startsWith('/') ? strippedHref : `/${strippedHref}`;
```

Comment:
I think we should change `sbStripSlugURL` to always return with the leading slash instead of trying to handle this condition outside of it everywhere else.

Reply:
Sure I can do that!